### PR TITLE
[Fix] Avoid scope switching when using mmdet inference interface

### PR DESCRIPTION
**Pull request by Ben-Louis**
## Motivation

While inferring top-down models on videos, the scope alternates between 'mmdet' and 'mmpose' repeatedly, which often results in numerous warning messages.
屏幕截图 2023-03-10 154340

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: mmpose/utils/config_utils.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmpose.utils.typing import ConfigDict
+
+
+def convert_mmdet_test_pipeline(cfg: ConfigDict) -> ConfigDict:
```
Comment (ly015) [inline mmpose/utils/config_utils.py:5]: What about naming the function 'port_mmdet_pipeline' or 'adapt_mmdet_pipeline'?

**Comment by ly015** [comment]
BTW, please also use lazy import for mmdetection in other parts of mmpose package, like `https://github.com/open-mmlab/mmpose/blob/dev-1.x/mmpose/apis/inferencers/pose2d_inferencer.py#L7`. So the model training/testing and document compiling will not depend on mmdetection.

**Comment by ly015** [comment]
> BTW, please also use lazy import for mmdetection in other parts of mmpose package, like `https://github.com/open-mmlab/mmpose/blob/dev-1.x/mmpose/apis/inferencers/pose2d_inferencer.py#L7`. So the model training/testing and document compiling will not depend on mmdetection.

Fixed in #2045 . No need to update this PR.

File: mmpose/utils/config_utils.py (hunk 1 above)
Comment (Ben-Louis) [reply mmpose/utils/config_utils.py:5]: Good idea. I think 'adapt_mmdet_pipeline' is preferable

**Review by ly015 — APPROVED** [review]

**Pull request merged**